demo-notebooks/guided-demos/0_basic_ray.ipynb

```diff
@@ -61,7 +61,8 @@
     "# Create and configure our cluster object\n",
     "# The SDK will try to find the name of your default local queue based on the annotation \"kueue.x-k8s.io/default-queue\": \"true\" unless you specify the local queue manually below\n",
     "cluster = Cluster(ClusterConfiguration(\n",
-    "    name='raytest',\n",
+    "    name='raytest', \n",
+    "    head_gpus=1, # For GPU enabled workloads set the head_gpus and num_gpus\n",
```

Comment:
```suggestion
    "    head_gpus=0, # For GPU enabled workloads set the head_gpus and num_gpus\n",
```
This nb didn't have GPUs in it by default so it is fine to leave it as is.

Reply:
Added head_gpus=0 by default as we have num_gpus=0 here as well